targets/TARGET_NXP/TARGET_LPC81X/TARGET_LPC810/device/TOOLCHAIN_ARM_MICRO/LPC810.sct

```diff
@@ -1,14 +1,48 @@
+#! armcc -E
 
-LR_IROM1 0x00000000 0x1000  {    ; load region size_region (4k)
-  ER_IROM1 0x00000000 0x1000  {  ; load address = execution address
+#if !defined(MBED_APP_START)
+  #define MBED_APP_START            0x00000000
+#endif
+
+; 4K flash
+#if !defined(MBED_APP_SIZE)
+  #define MBED_APP_SIZE             0x1000
+#endif
+
+; 1KB 
+#if !defined(MBED_RAM_START)
+  #define MBED_RAM_START            0x10000000
+#endif
+
+#if !defined(MBED_RAM_SIZE)
+  #define MBED_RAM_SIZE             0x400
+#endif
+
+
+#if !defined(MBED_BOOT_STACK_SIZE)
+  #define MBED_BOOT_STACK_SIZE      0x400
```

Comment:
This SoC has very limited RAM, this is making the whole RAM as Stack. A smaller default value?

Reply:
That was my concern as well, default on Mbed 2 stack size is not 0x400 but 0x1000. Even if we reduce the value, I am not sure if the blinky will also work on device with 1K RAM. For now we can overwrite the stack size explicitly in linker, but we need some long term solution for devices with <8K RAM 

CC @maclobdell